controller/system_backup_controller.go

```diff
@@ -472,6 +485,12 @@ func (c *SystemBackupController) reconcile(name string, backupTargetClient engin
 	return nil
 }
 
+func systemBackupInFinalState(sb *longhorn.SystemBackup) bool {
+	return sb.Status.State == longhorn.SystemBackupStateDeleting ||
```

Comment:
nit: add a comment for explaining why we regard `SystemBackupStateDeleting` as a final state here.

Reply:
Added. Thanks